pkg/chartvalues/aws_operator_template.go

```diff
@@ -33,6 +33,8 @@ const awsOperatorTemplate = `Installation:
         Route53:
           Enabled: true
         Encrypter: '{{ .Provider.AWS.Encrypter }}'
+        TrustedAdvisor:
+          Enabled: {{ .Provider.AWS.TrustedAdvisor.Enabled }}
```

Comment:
I think the configuration isn't that useful. As you wrote it's  always true. I'd vote for replacing this line with `Enabled: true` and don't allow configuring it from the outside. Otherwise it's only confusing and scary to remove in the future. In general I believe we should expose settings as they are needed and not just in case.

Reply:
> it's always true

It is not, which is the whole point of addressing the PM.

Reply:
It's always true in our test environments.

Reply:
Is it though? Why is that? I would have rather expected it to be always false there, because why should we pay for such a service in test envs?

Reply:
I took that knowledge from the PR desc.

Reply:
Yeah and then I studied a bit more and noticed it's actually always false. AWS console is a bit confusing for TA and there's not so much documentation for it. But I agree with Pawel that hardcoded value here should be fine as it's always what it is. The service is global in AWS (AFAICT) and we don't have the full support there so we lack the API access for it.